src/services/orderbook_service.ts

```diff
@@ -128,8 +133,31 @@ export class OrderBookService {
         const { fresh, expired } = orderUtils.groupByFreshness(apiOrders, SRA_ORDER_EXPIRATION_BUFFER_SECONDS);
         alertOnExpiredOrders(expired);
 
-        // Post-filters
-        const filteredApiOrders = orderUtils.filterOrders(fresh, ordersFilterParams);
+        // Join with persistent orders
+        let persistentOrders: APIOrderWithMetaData[] = [];
+        if (ordersFilterParams.unfillable === true) {
```

Comment:
My only thought here is we want to avoid sending too much data, to prevent ourselves from being taken down.

We might want to enforce that `unfillable` can only be used in conjunction when `maker_address`  is filtered as well.

Or what ever the most common behaviour is expected.

I have concern of just an `GET /sra/orders?unfillable=true` returning large payloads.

Reply:
doneskiii